Fails some tests:

```
======================================================================
FAIL: test_grouping (miro.test.infolisttest.InfoListDataTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/z3p/Projects/miro/tv/linux/dist/usr/lib/python2.7/site-packages/miro/test/infolisttest.py", line 259, in test_grouping
    self.check_update_grouping(first_letter_grouping)
  File "/home/z3p/Projects/miro/tv/linux/dist/usr/lib/python2.7/site-packages/miro/test/infolisttest.py", line 146, in check_update_grouping
    self.check_grouping()
  File "/home/z3p/Projects/miro/tv/linux/dist/usr/lib/python2.7/site-packages/miro/test/infolisttest.py", line 160, in check_grouping
    (i, total_count))
AssertionError: Tuples differ: (0, 2, FakeInfo('miro', 0)) != (0, 2)

First tuple contains 1 additional elements.
First extra element 2:
FakeInfo('miro', 0)

- (0, 2, FakeInfo('miro', 0))
+ (0, 2)

======================================================================
FAIL: test_grouping (miro.test.infolisttest.InfoListGTKTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/z3p/Projects/miro/tv/linux/dist/usr/lib/python2.7/site-packages/miro/test/infolisttest.py", line 259, in test_grouping
    self.check_update_grouping(first_letter_grouping)
  File "/home/z3p/Projects/miro/tv/linux/dist/usr/lib/python2.7/site-packages/miro/test/infolisttest.py", line 146, in check_update_grouping
    self.check_grouping()
  File "/home/z3p/Projects/miro/tv/linux/dist/usr/lib/python2.7/site-packages/miro/test/infolisttest.py", line 160, in check_grouping
    (i, total_count))
AssertionError: Tuples differ: (0, 2, FakeInfo('miro', 0)) != (0, 2)

First tuple contains 1 additional elements.
First extra element 2:
FakeInfo('miro', 0)

- (0, 2, FakeInfo('miro', 0))
+ (0, 2)

----------------------------------------------------------------------
```

--- a/tv/lib/test/infolisttest.py
+++ b/tv/lib/test/infolisttest.py
@@ -153,18 +153,19 @@ class InfoListTestBase(MiroTestCase):
             # key is the value that new_grouping() returned.  group is an
             # iterater that contains infos in that group
             group = list(group)
+            first_info = group[0]
             total_count = len(group)
             for i, info in enumerate(group):
                 # check get_group_info()
                 self.assertEquals(self.infolist.get_group_info(info.id),
-                        (i, total_count))
+                        (i, total_count, first_info))
                 # check get_group_top()
                 self.assertEquals(self.infolist.get_group_top(info.id),
-                        group[0])
+                                  first_info)
                 # check group_info part of row_for_iter
                 it = self.infolist.iter_for_id(info.id)
                 info, attrs, group_info = self.infolist.row_for_iter(it)
-                self.assertEquals(group_info, (i, total_count))
+                self.assertEquals(group_info, (i, total_count, first_info))
 
 class InfoListDataTest(InfoListTestBase):
     def check_list(self, *names):

